### DISCUSSION TITLE
Where to place  visitor map widget to show on bottom of home page?

### DISCUSSION THREAD

**Discussion by shashikantilager** [alshedivat/al-folio#1895] [Q&A] [answered]
Hi all,

I want to insert a javascript of the visitor map widget (https://mapmyvisitors.com). When I place the script at the end of  About.md page, it renders before the news item. Where should I keep the map widget script so that it will be shown at the bottom of the page? Please advise.

**Comment by george-gca** [comment, marked as answer]
The `about.md` page uses `layout: about`. If you go to [_layouts/about.html](https://github.com/alshedivat/al-folio/blob/master/_layouts/about.html) you'll see that it uses `layout: default`, which is in [_layouts/default.html](https://github.com/alshedivat/al-folio/blob/master/_layouts/default.html). So your javascript should be in one of them, depending where you'll want it to be displayed. Just note that almost all pages end up using `layout: default`.

**Reply by shashikantilager** [reply]
Thanks a ton!
